WP_Auth0.php

```diff
@@ -2,13 +2,13 @@
 /**
  * Plugin Name: Login by Auth0
  * Description: Login by Auth0 provides improved username/password login, Passwordless login, Social login and Single Sign On for all your sites.
- * Version: 3.8.1
+ * Version: 3.9.0-beta
```

Comment:
First change after a release needs to bump the version number in `master` in a way that can be updated when the official release comes out.